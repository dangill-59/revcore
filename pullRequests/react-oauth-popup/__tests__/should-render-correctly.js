import OauthPopup from '../src/index.js';
import React from 'react';
import renderer from 'react-test-renderer';

test('OauthPopup component renders correctly', () => {
  const tree = renderer
    .create(
      <OauthPopup>
        <div>coooooooool</div>
        <h2>siiiiiiiick</h2>
        <h2>leeegiiiiit</h2>
        <button>Doooooope</button>
      </OauthPopup>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
