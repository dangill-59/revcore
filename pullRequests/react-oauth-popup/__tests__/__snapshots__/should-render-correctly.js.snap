// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OauthPopup component renders correctly 1`] = `
<div
  onClick={[Function]}
>
   
  <div>
    coooooooool
  </div>
  <h2>
    siiiiiiiick
  </h2>
  <h2>
    leeegiiiiit
  </h2>
  <button>
    Doooooope
  </button>
   
</div>
`;
